conferences/presentations/RIghtsCon-2021-06-07/script.md

```diff
@@ -0,0 +1,55 @@
+In particular, we will demonstrate the Turing Way guide for Ethical Research, the community behind its development, and its human rights-centred approach to data science ethics as a practice that includes activism for change. 
+Most importantly, we will demonstrate how RightsCon attendees can get involved!
+
+(based heavily on [this talk](https://zenodo.org/record/3968454#.YIA1Vmgo-i5) by Malvika Sharan, which was 10 minutes.
+
+## Skeleton:
+1. Intros and what is the Turing Way? (2 mins)
+3. What is the Turing Way Guide for Ethical Research? (5 mins)
+4. How can you get involved? (3 mins)
+
+## The Turing Way: Building a collaborative guide to ethical data science
+### Presenters: Malvika Sharan and Laura Carter
+
+(Introduce speakers)
+
+The Turing Way is a collaborative book project.
+It is based at the Alan Turing Institute, the UK's national centre for data science and artificial intelligence. 
+It's an open source, collaborative project: and has contributors from all over the world.
+
+The aim of the Turing Way is to make reproducible, ethical and inclusive data science 'too easy not to do.'
+
+There are five books in the Turing Way: covering collaboration, project design, reproducibility, communication and outreach, and ethics.
+We're going to focus on the Guide to Ethical Research.
+
+The aim of the Guide to Ethical Research is to support data scientists - and researchers using data science - to work in an ethical way.
+We start from the position that as data science researchers, we have an obligation to consider the impact of our research. 
+So far, contributors to the Guide to Ethical Research have contributed chapters on working with institutional ethics boards, the relationship between ethics and research integrity, and ethics in preclinical research.
+
+But we don't see ethics as a static or one-off part of research.
+Ethical research includes every part of research: including making sure your institution is treating its worker, its research participants, and everyone else involved, well. 
+The Guide to Ethical Research also includes information on how to advocate for policy change within your institution: if you don't like how something is done, you can change it!
+It also covers how to be an activist researcher, and how and why unionisation can be important for data scientists to practice ethically.
+
+The Guide to Ethical Research is a work in progress.
+Current chapters-in-progress include how ethical data science fits into legal, policy and human rights frameworks; how to address and overcome challenges in ethical practice; and participatory science methods.
+
+We'd like to invite you to become a contributor to the Turing Way, to help us develop these chapters, and many more. 
+You don't need to have coding experience: just an idea and a willingness to learn. 
```

Comment:
Could you add in a point about case studies and lived experience being valuable as well?